### DISCUSSION TITLE
how to change the built-in style of html?

### DISCUSSION THREAD

**Discussion by ibearye** [tailwindlabs/tailwindcss#4312] [Help]

**Comment by dance2die** [comment]
`@tailwind base;`'s [preflight core plugin](https://tailwindcss.com/docs/configuration#core-plugins) resets browser styles unless turned off in `tailwind.config.js`.

If you want to change styles for DOM elements (such as `h1/h2...`, `p`, `div`, etc), then you can change "base" styles as mentioned here.: https://tailwindcss.com/docs/adding-base-styles#using-css

Demo: https://play.tailwindcss.com/uQi2i9KwwR?file=css
_Your Tailwind CSS file._

```css
@tailwind base;
@tailwind components;
@tailwind utilities;

@layer base {
  h1 {
    @apply text-5xl font-bold text-pink-500;
  }
  h2 {
    @apply text-3xl font-serif text-light-blue-500;
  }
}
```
